doc/changes/changes_0.1.0.md

```diff
@@ -1,8 +1,15 @@
-# transformers-extension 0.1.0, released XXXX-XX-XX
+# Transformers Extension 0.1.0, released 2022-09-23
 
+Code name: Add a downloader UDF and a set of prediction UDFs using the transformers API
 
-## Summary
 
+## Summary
+This is the initial release of the transformers-extension which provides a 
```

Comment:
```suggestion
## Summary

This is the initial release of the transformers-extension which provides a 
```

The original Markdown spec requires a blank line between a headline and a text body. Most parsers are not that picky though. :-)